src/main/java/com/mongodb/jdbc/MongoConnection.java

```diff
@@ -376,14 +324,7 @@ public SQLXML createSQLXML() throws SQLException {
                 Thread.currentThread().getStackTrace()[1].toString());
     }
 
-    private void validateConn() throws SQLException {
```

Comment:
We don't need to make this abstract. We just need to change `"SELECT 1 from DUAL" to "SELECT 1"`, since that works in MySQL.

Reply:
Oh right, I must have breezed past that part of the design when going through and checking off methods. I'll address this.

Reply:
Done